pom.xml

```diff
@@ -30,46 +30,46 @@
 		<calculationVersion>1.3.0-SNAPSHOT</calculationVersion>
-		<serialization.xstreamVersion>0.2.16-SNAPSHOT</serialization.xstreamVersion>
-		<htmlwidgetsVersion>1.11.0-SNAPSHOT</htmlwidgetsVersion>
-		<reportingVersion>1.23.0-SNAPSHOT</reportingVersion>
-		<reportingRestVersion>1.13.0-SNAPSHOT</reportingRestVersion>
+		<serialization.xstreamVersion>0.2.15</serialization.xstreamVersion>
+		<htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
+		<reportingVersion>1.22.0</reportingVersion>
+		<reportingRestVersion>1.12.0</reportingRestVersion>
 		<metadatadeployVersion>1.13.0-SNAPSHOT</metadatadeployVersion>
 		<metadatasharingVersion>1.8.0-SNAPSHOT</metadatasharingVersion>
 		<metadatamappingVersion>1.4.0-SNAPSHOT</metadatamappingVersion>
-		<uicommonsVersion>2.18.0-SNAPSHOT</uicommonsVersion>
-		<appuiVersion>1.15.0-SNAPSHOT</appuiVersion>
-		<htmlformentryVersion>4.0.2-SNAPSHOT</htmlformentryVersion>
+		<uicommonsVersion>2.18.0</uicommonsVersion>
+		<appuiVersion>1.14.0</appuiVersion>
+		<htmlformentryVersion>4.0.1</htmlformentryVersion>
 		<htmlformentryuiVersion>2.0.0-SNAPSHOT</htmlformentryuiVersion>
 		<coreappsVersion>1.31.2-SNAPSHOT</coreappsVersion>
 		<webservices.restVersion>2.31.0-SNAPSHOT</webservices.restVersion>
 		<referencedemodataVersion>1.4.7-SNAPSHOT</referencedemodataVersion>
 		<dataexchangeVersion>1.3.5-SNAPSHOT</dataexchangeVersion>
-		<allergyuiVersion>1.8.5-SNAPSHOT</allergyuiVersion>
-		<formentryappVersion>1.4.3-SNAPSHOT</formentryappVersion>
+		<allergyuiVersion>1.8.4</allergyuiVersion>
+		<formentryappVersion>1.4.2</formentryappVersion>
 		<atlasVersion>2.2.5-SNAPSHOT</atlasVersion>
-		<appointmentschedulingVersion>1.15.0-SNAPSHOT</appointmentschedulingVersion>
-		<appointmentschedulinguiVersion>1.11.0-SNAPSHOT</appointmentschedulinguiVersion>
-		<namephoneticsVersion>1.18.0-SNAPSHOT</namephoneticsVersion> <!-- The omod for namephonetics is not included in the distro, but we keep the version around since it's a semi-supported add-on -->
-		<adminuiVersion>1.6.0-SNAPSHOT</adminuiVersion>
+		<appointmentschedulingVersion>1.14.0</appointmentschedulingVersion>
+		<appointmentschedulinguiVersion>1.10.0</appointmentschedulinguiVersion>
+		<namephoneticsVersion>1.18.0</namephoneticsVersion> <!-- The omod for namephonetics is not included in the distro, but we keep the version around since it's a semi-supported add-on -->
+		<adminuiVersion>1.5.0</adminuiVersion>
 		<uitestframeworkVersion>2.4.0-SNAPSHOT</uitestframeworkVersion>
-		<legacyuiVersion>1.8.4-SNAPSHOT</legacyuiVersion>
-		<reportingcompatibilityVersion>2.0.9-SNAPSHOT</reportingcompatibilityVersion>
+		<legacyuiVersion>1.8.3</legacyuiVersion>
+		<reportingcompatibilityVersion>2.0.8</reportingcompatibilityVersion>
 		<fhirVersion>1.3.0-SNAPSHOT</fhirVersion>
 		<owaVersion>1.13.0-SNAPSHOT</owaVersion>
 		<reportinguiVersion>1.8.0-SNAPSHOT</reportinguiVersion>
-		<addresshierarchyVersion>2.15.0-SNAPSHOT</addresshierarchyVersion>
```

Comment:
why not 2.15.0 , any reason why you reverted to 2.14.2

Reply:
https://wiki.openmrs.org/display/docs/Versioning

Reply:
thanks @dkayiwa